portal/app/core/assets/js/simple-diagram.js

```diff
@@ -629,8 +629,16 @@
         		.attr('y', coords.y);
         }
 
+		if (opts.shape === 'hvac') {
```

Comment:
The formatting in this looks off in GitHub. Is this an issue with your editor? What is going on?

Reply:
Fixed. I think there was inconsistency with use of indents and spaces on different lines that did not show on either of my text editors (atom and text wrangler).